templates/tools/dockerfile/test/sanity/Dockerfile.template

```diff
@@ -14,16 +14,12 @@
   # See the License for the specific language governing permissions and
   # limitations under the License.
   
-  <%include file="../../python_debian10.include"/>
+  <%include file="../../python_debian11.include"/>
```

Comment:
it looks like after this, python_debian10.include becomes unused - please check if that's the case and remove it if so.

Reply:
Good point. Done!